### DISCUSSION TITLE
How is the TreeView event defined

### DISCUSSION THREAD

**Discussion by Hub-moon** [AvaloniaUI/Avalonia#13221] [General]
AvaloniaUI is TreeView does not have the same events as WPF, such as SelectedItemChanged

**Comment by pyking-wang** [comment]
```
this.WhenAnyValue(x => x.SelectedItem)
.Subscribe(x =>
{
    ChangedEvent;
});
```

**Reply by Hub-moon** [reply]
Thank you,I found another way to do this, through the selectedNodes.collectionChanged event
